pkg/types/ovirt/platform.go

```diff
@@ -0,0 +1,31 @@
+package ovirt
+
+// Platform stores all the global configuration that all
+// machinesets use.
+type Platform struct {
```

Comment:
each field in this struct must have a field comment that explains what this field is for, provide validation information, defaults etc..

Reply:
done